packages/react-topology/src/components/contextmenu/ContextSubMenuItem.scss

```diff
@@ -0,0 +1,11 @@
+.topology-context-sub-menu {
```

Comment:
This SASS file is unused. I'd just compile it to CSS since it seems to be the only SASS file.

Reply:
sass is not used, I moved the styles to the react-styles